packages/browser-integration-tests/suites/public-api/captureException/emptyObj/test.ts

```diff
@@ -12,7 +12,7 @@ sentryTest('should capture an empty object', async ({ getLocalTestPath, page })
   expect(eventData.exception?.values).toHaveLength(1);
   expect(eventData.exception?.values?.[0]).toMatchObject({
     type: 'Error',
-    value: 'Non-Error exception captured with keys: [object has no keys]',
+    value: 'Object captured as exception with keys: [object has no keys]',
```

Comment:
Wondering if this should just read `'Object captured as exception'`. Probably fine to save bundle.

Reply:
I originally wanted to do that as well, but the `[object has no keys]` part comes from the existing `extractExceptionKeysForMessage` which is used in multiple places, so I figured we can just keep it?

Reply:
Yea let's keep it. I trust people to understand what's going on here.